packages/pigeon/platform_tests/test_plugin/windows/test/null_fields_test.cpp

```diff
@@ -189,32 +194,41 @@ TEST_F(NullFieldsTest, ReplyToMapWithValues) {
   reply.set_request(request);
   reply.set_type(NullFieldsSearchReplyType::success);
 
-  EncodableMap map = MapFromReply(reply);
+  const EncodableList wrapped = ListFromReply(reply);
 
-  EXPECT_EQ(map.size(), 5);
-  EXPECT_EQ(*ExpectAndGet<std::string>(map, "result"), "result");
-  EXPECT_EQ(*ExpectAndGet<std::string>(map, "error"), "error");
-  const EncodableList& indices = *ExpectAndGet<EncodableList>(map, "indices");
+  EXPECT_EQ(wrapped.size(), 1);
+
+  const EncodableList list = *ExpectAndGetIndex<EncodableList>(wrapped, 0);
+
+  EXPECT_EQ(list.size(), 5);
+  EXPECT_EQ(*ExpectAndGetIndex<std::string>(list, 0), "result");
+  EXPECT_EQ(*ExpectAndGetIndex<std::string>(list, 1), "error");
+  const EncodableList& indices = *ExpectAndGetIndex<EncodableList>(list, 2);
   EXPECT_EQ(indices.size(), 3);
   EXPECT_EQ(indices[0].LongValue(), 1L);
   EXPECT_EQ(indices[1].LongValue(), 2L);
   EXPECT_EQ(indices[2].LongValue(), 3L);
-  const EncodableMap& request_map = *ExpectAndGet<EncodableMap>(map, "request");
-  EXPECT_EQ(*ExpectAndGet<std::string>(request_map, "query"), "hello");
-  EXPECT_EQ(*ExpectAndGet<int>(map, "type"), 0);
+  const EncodableList& request_list =
+      *ExpectAndGetIndex<EncodableList>(list, 4);
+  EXPECT_EQ(*ExpectAndGetIndex<std::string>(request_list, 0), "hello");
+  EXPECT_EQ(*ExpectAndGetIndex<int>(request_list, 1), 0);
 }
 
-TEST_F(NullFieldsTest, ReplyToMapWithNulls) {
+TEST_F(NullFieldsTest, ReplyToListWithNulls) {
   NullFieldsSearchReply reply;
 
-  EncodableMap map = MapFromReply(reply);
+  const EncodableList wrapped = ListFromReply(reply);
+
+  EXPECT_EQ(wrapped.size(), 1);
+
+  const EncodableList list = *ExpectAndGetIndex<EncodableList>(wrapped, 0);
 
-  EXPECT_EQ(map.size(), 5);
-  EXPECT_TRUE(map[EncodableValue("result")].IsNull());
-  EXPECT_TRUE(map[EncodableValue("error")].IsNull());
-  EXPECT_TRUE(map[EncodableValue("indices")].IsNull());
-  EXPECT_TRUE(map[EncodableValue("request")].IsNull());
-  EXPECT_TRUE(map[EncodableValue("type")].IsNull());
+  EXPECT_EQ(list.size(), 5);
+  EXPECT_TRUE(list[0].IsNull());
+  EXPECT_TRUE(list[1].IsNull());
+  EXPECT_TRUE(list[2].IsNull());
+  EXPECT_TRUE(list[3].IsNull());
+  EXPECT_TRUE(list[4].IsNull());
```

Comment:
Optional: since this test is asserting that everything is null, you could do:
```
const int field_count = 5;
EXPECT_EQ(list.size(), field_count);
for (int i = 0; i < field_count; ++i) {
  EXPECT_TRUE(list[i].IsNull());
}
```